templates/http/modules.j2

```diff
@@ -330,3 +330,26 @@ sub_filter_types {{ sub_filter['types'] if sub_filter['types'] is string else su
 {% endif %}
 
 {% endmacro %}
+
+{# NGINX HTTP v2 -- ngx_http_v2_module #}
+{% macro http2(http2, scope='http') %}
+{% if http2 is defined %}
+{% if scope != 'location' %} {# 'http' and 'server' contexts #}
```

Comment:
Leaving an empty space between the Jinja2 templated line and the Jinja2 comment will lead to some "undesired" white space in the NGINX config
```suggestion
{% if scope != 'location' %}{# The following scoped directives are not available in the 'location' context #}
```